assets/src/edit-story/components/panels/design/borderRadius/borderRadius.js

```diff
@@ -223,11 +224,13 @@ function BorderRadiusPanel({ selectedElements, pushUpdateForObject }) {
         </InputContainer>
 
         <LockContainer>
-          <LockToggle
-            isLocked={borderRadius.locked}
-            onClick={() => handleLockChange(!borderRadius.locked)}
-            aria-label={__('Toggle corner radius lock', 'web-stories')}
-          />
+          <Tooltip title={__('Toggle identical radii', 'web-stories')}>
```

Comment:
The tests are failing now because the label was used, however, it feels like we should still use simpler words, such as "border radius" instead of "radii". Also, I think that "lock" is quite commonly used in design tools.

Why change the wording?

Reply:
Maybe UX can consult here based on what's generally used in design tools.